### DISCUSSION TITLE
CVSS 4.0

### DISCUSSION THREAD

**Discussion by dor-hayun** [aquasecurity/trivy#5404] [Ideas]
### Description

Hi,
Are you willing to import the CVSS 4.0 data (score+vector) to your DB?

### Target

Container Image

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
It depends on the data source.
https://aquasecurity.github.io/trivy/v0.46/docs/scanner/vulnerability/

**Comment by knqyf263** [comment]
Created https://github.com/aquasecurity/trivy/issues/7059
